etc/less.profile

```diff
@@ -16,6 +16,7 @@ novideo
 shell none
 tracelog
 
+private-bin less
```

Comment:
Appropriate compression tools are required if you're viewing compressed files directly.